.github/workflows/optimize-release-optimize-c8-only.yml

```diff
@@ -0,0 +1,301 @@
+          echo "DOCKER_LATEST: ${{ inputs.DOCKER_LATEST }}"
+          echo "IS_DRY_RUN: ${{ inputs.IS_DRY_RUN }}"
+
+      - name: "Read Java / Version Info"
+        id: "pom-info"
+        uses: YunaBraska/java-info-action@main
+
+      - name: Setup Maven
+        uses: ./.github/actions/setup-maven
+        with:
+          secrets: ${{ toJSON(secrets) }}
+
+      - name: Is current release major or minor.
+        id: release-type
+        env:
+          RELEASE_VERSION: ${{ github.event.inputs.RELEASE_VERSION || '8.6.0-alpha3-rc3' }}
+        run: |
+          is_major_or_minor="false"
+          is_patch="true"
+          is_rc="true"
+          is_alpha="true"
+          patch_version=$(echo "$RELEASE_VERSION" | cut -d. -f3)
+          if [[ ! "$patch_version" == *"-"* ]] && [[ "$patch_version" == "0" ]]; then
+            is_major_or_minor="true"
+            is_patch="false"
+            echo "is_patchr=$is_patch"
+            echo "is_major_or_minor=$is_major_or_minor"
+          fi
+          if [[ ! $RELEASE_VERSION =~ rc[0-9]+$ ]]; then
+            is_rc="false"
+          fi
+          if [[ ! $RELEASE_VERSION =~ alpha[0-9]+$ ]]; then
+            is_alpha="false"
+          fi
+          echo "is_major_or_minor=$is_major_or_minor" >> "$GITHUB_OUTPUT"
+          echo "is_patch=$is_patch" >> "$GITHUB_OUTPUT"
+          echo "is_rc=$is_rc" >> "$GITHUB_OUTPUT"
+          echo "is_alpha=$is_alpha" >> "$GITHUB_OUTPUT"
+
+      - name: Install common tooling (buildx) # required on self-hosted runners
+        uses: camunda/infra-global-github-actions/common-tooling@main
+        with:
+          buildx-enabled: true
+          java-enabled: false
+          node-enabled: false
+          python-enabled: false
+          yarn-enabled: false
+
+      - name: Define common variables
+        id: define-values
+        uses: ./.github/actions/git-environment
+
+      - name: Expose common variables as Env
+        run: |
+          {
+            echo "DOCKER_LATEST=${{ github.event.inputs.DOCKER_LATEST || false}}"
+            echo "DEVELOPMENT_VERSION=${{ github.event.inputs.DEVELOPMENT_VERSION || '8.6.0-SNAPSHOT' }}"
+            echo "RELEASE_VERSION=${{ github.event.inputs.RELEASE_VERSION || '8.6.0-alpha3-rc3' }}"
+            echo "BRANCH=${{ github.event.inputs.BRANCH || 'test-optimize-release' }}"
+            echo "REVISION=${{ steps.define-values.outputs.git_commit_hash }}"
+            echo "MAJOR_OR_MINOR=${{ steps.release-type.outputs.is_major_or_minor }}"
+            echo "IS_PATCH=${{ steps.release-type.outputs.is_patch }}"
+            echo "IS_ALPHA=${{ steps.release-type.outputs.is_alpha }}"
+            echo "IS_RC=${{ steps.release-type.outputs.is_rc }}"
+            echo "IS_DRY_RUN=${{ github.event.inputs.IS_DRY_RUN || 'false' }}"
+            echo "TAG=${{ github.event.inputs.RELEASE_VERSION || '8.6.0-alpha3-rc3' }}-optimize-test"
+          } >> "$GITHUB_ENV"
+
+      - name: Login to prod Harbor docker registry
```

Comment:
Do we need this?

Reply:
I believe this is used in the smoke test to pull the docker image

Reply:
ah, alright! that makes sense